src/coverage-system/coverageservice.ts

```diff
@@ -9,6 +9,7 @@ import {
 } from "vscode";
 
 import {Config} from "../extension/config";
+import { StatusBarToggler } from "../extension/statusbartoggler";
 import {CoverageParser} from "../files/coverageparser";
```

Comment:
super minor but we probably want these to all include spaces around the brackets (not sure why the linter didn't figure this out...)

Reply:
Agreed - my VSCode user-specific formatting settings were interfering with the project specific ones. I'll likely push a separate "linted" branch/PR (I hate mixing logic with formatting)